frameworks/Nim/mofuw/mofuw.dockerfile

```diff
@@ -14,17 +14,15 @@ RUN apt update -yqq &&                                                  \
     ./koch nimble &&                                                    \
     ./koch tools
 
-ENV PATH=$PATH:/nim/nim-devel/bin:/root/.nimble/bin
+ENV PATH $PATH:/nim/nim-devel/bin:/root/.nimble/bin
 
+WORKDIR /mofu_framework
```

Comment:
Is this WOKDIR necessary?

Reply:
As far as I know, there's no functional difference.  The intent is to make it clear where the framework is being installed, and to show that where/how it's installed does not depend on a working directory inherited from earlier steps in the dockerfile.